test/unit/plugins/doca_memos/doca_memos_backend_test.cpp

```
@@ -0,0 +1,1747 @@
+    void
+    TearDown() override {
+        // Clean up engine if created
+        if (engine_) {
+            engine_.reset();
+        }
+    }
+
+    void
+    setupInitParams(bool enable_progress_thread = false,
+                    nixl_thread_sync_t sync_mode = nixl_thread_sync_t::NIXL_THREAD_SYNC_NONE,
+                    std::string device_name = "/dev/nvme0n1",
```

Cppcheck (2.20.0)

[performance] Function parameter 'device_name' should be passed by const reference.

(passedByValue)